posthog/hogql_queries/web_analytics/revenue_example_events_query_runner.py

```diff
@@ -62,10 +62,13 @@ def to_query(self) -> ast.SelectQuery:
                 ),
                 ast.Field(chain=["event"]),
                 ast.Alias(
-                    alias="original_revenue", expr=revenue_expression(tracking_config, do_currency_conversion=False)
+                    alias="original_revenue",
+                    expr=revenue_expression_for_events(tracking_config, do_currency_conversion=False),
                 ),
-                ast.Alias(alias="revenue", expr=revenue_expression(tracking_config, self.do_currency_conversion)),
-                ast.Alias(alias="original_currency", expr=revenue_currency_expression(tracking_config)),
+                ast.Alias(
+                    alias="revenue", expr=revenue_expression_for_events(tracking_config, self.do_currency_conversion)
+                ),
+                ast.Alias(alias="original_currency", expr=currency_expression_for_all_events(tracking_config)),
```

Comment:
I'd be tempted to switch the order here, so it's original revenue, original currency, revenue, currency. I think it'll be easier to read as a pair of (revenue, currency)

Reply:
Yeah, that makes sense, let me do that :)